api/src/test/scala/com/gu/adapters/http/AvatarServletTests.scala

```diff
@@ -21,10 +21,12 @@ class AvatarServletTests extends TestHelpers {
   // See https://issues.scala-lang.org/browse/SI-5091
   val apiUrl: String = avatarServletProps.apiUrl
   val apiKey = avatarServletProps.apiKeys.head
+  val avatarStore = AvatarStore(new TestFileStore(storeProps.fsProcessedBucket), new TestKVStore(storeProps.kvTable), storeProps)
 
   addServlet(
     new AvatarServlet(
-      AvatarStore(new TestFileStore(storeProps.fsProcessedBucket), new TestKVStore(storeProps.kvTable), storeProps),
+      avatarStore,
+      //avatarUpdateService,
```

Comment:
what does this mean?

Reply:
Ah that's some code that got removed but incompletely! I'll remove it thanks.